src/Compilers/CSharp/Portable/Emitter/EditAndContinue/CSharpSymbolMatcher.cs

```diff
@@ -498,7 +498,7 @@ public override Symbol VisitNamedType(NamedTypeSymbol sourceType)
                         return null;
                     }
 
-                    return TupleTypeSymbol.Create(otherDef, sourceType.TupleElementNames);
+                    return otherDef;
```

Comment:
I could not find an idea for the comment. It seems like a simplification change. Do you have ideas for the comment?

---
In reply to: [116290923](https://github.com/dotnet/roslyn/pull/19283#discussion_r116290923) [](ancestors = 116290923)